package.xml

```diff
@@ -16,7 +16,7 @@
 -->
 <package format="2">
   <name>carma_base</name>
-  <version>3.1.0</version>
+  <version>3.1.1</version>
```

Comment:
Checked with @TonysCousin, he suggests we use 3.2.0 instead of 3.1.1. Minor version numbers are for defect fixes and this is part of an intentional enhancement.

Reply:
This is just a resync to master for develop. Then I'll put up the actual PR